Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -1791,6 +1812,21 @@ public JSONObject getLatestReferringParams() {
         return latestParams;
     }
 
+    public JSONObject getLatestReferringParamsSync() {
```

Comment:
Please add comments to this method with strict caveat on calling this from UI thread